### ISSUE TITLE
Focus trap Doesn't work on (dev) React strict mode

### ISSUE THREAD

**Issue by ynifamily3** [focus-trap/focus-trap-react#704]
React Version: 18.1.0

The focus trap doesn't seem to work at all. If you disable strict mode or build it, it works normally.

**Comment by stefcameron** [comment]
Thanks. I believe this will be resolved by fixing #77

**Comment by stefcameron** [comment]
Hoping #710 will fix this.

**Comment by stefcameron** [comment]
Look for a fix to this (I hope!!) in v9.0.0 being published right now.
